tests/test_integration.py

```diff
@@ -92,23 +99,57 @@ def testBasicAuthRoutes(self):
     def testHome(self, mockUser, mockTemplate):
 
         mockTemplate.return_value = "test"
+        mockUser.query.filter_by.return_value.first.return_value = self.testUser
 
         with self.client.post(VERSION_ROOT + '/home/', data=dict(username="", password="")):
             mockTemplate.assert_called_with(
                 'home.html', clients=None, message='Please Fill In Both Username and Password.', user=None)
+
         with self.client.post(VERSION_ROOT + '/home/', data=dict(username="steve", password="wrongpassword")):
             mockTemplate.assert_called_with(
                 'home.html', clients=None, message='Invalid Password. Try Again.', user=None)
+
         with self.client.post(VERSION_ROOT + '/home/', data=dict(username="steve", password="password")) as rv:
-            self.assertEqual(rv.status_code, 200)
+            self.assertEqual(rv.status_code, 302)
+
         mockUser.query.filter_by.return_value.first.return_value = None
         with self.client.post(VERSION_ROOT + '/home/', data=dict(username="steve", password="password")):
             mockTemplate.assert_called_with(
                 'home.html', clients=None, message='That username is not recognised. Please signup.', user=None)
 
-    @mock.patch("nmosauth.auth_server.security_api.authorization")
-    def testTokenEndpoint(self, mockAuthServer):
-        mockAuthServer.create_endpoint_response.return_value = BEARER_TOKEN
+    def testRegisterClient(self):
```

Comment:
Just a style comment, most of the other repos with test use `lowercase_separated_by_underscores` for the test methods. Can't see anything in the ways of working to specify a preference.

Reply:
Yeah they should be underscores / snake_case really according to PEP guidelines...